### ISSUE TITLE
Cisco APs to Victorialogs is crashing "runtime error: index out of range [0] with length 0"

### ISSUE THREAD

**Issue by gesnaud** [VictoriaMetrics/VictoriaMetrics#6548]
### Describe the bug

When I configure the Cisco's CBW150AX controller to send logs to Victorialogs and click on apply, Victorialogs is crashing with this crash stack:
```bash
# docker compose -f logs-playbook.yml logs victorialogs -f                                                                                          
victorialogs  | 2024-06-28T10:59:24.732Z        info    VictoriaMetrics/lib/logger/flag.go:12   build version: victoria-logs-20240627-130708-tags-v0.24.0-victorialogs-0-g7c8c040502                               
victorialogs  | 2024-06-28T10:59:24.732Z        info    VictoriaMetrics/lib/logger/flag.go:13   command-line flags                                                                                                 
victorialogs  | 2024-06-28T10:59:24.732Z        info    VictoriaMetrics/lib/logger/flag.go:20     -httpListenAddr=":9428"                                                                                          
victorialogs  | 2024-06-28T10:59:24.732Z        info    VictoriaMetrics/lib/logger/flag.go:20     -loggerLevel="INFO"                                                                                              
victorialogs  | 2024-06-28T10:59:24.733Z        info    VictoriaMetrics/lib/logger/flag.go:20     -storageDataPath="/vlogs"                                                                                        
victorialogs  | 2024-06-28T10:59:24.733Z        info    VictoriaMetrics/lib/logger/flag.go:20     -syslog.listenAddr.udp=":514"                                                                                    
victorialogs  | 2024-06-28T10:59:24.733Z        info    VictoriaMetrics/lib/logger/flag.go:20     -syslog.timezone="Europe/Paris"                                                                                  
victorialogs  | 2024-06-28T10:59:24.733Z        info    VictoriaMetrics/app/victoria-logs/main.go:42    starting VictoriaLogs at "[:9428]"...                                                                      
victorialogs  | 2024-06-28T10:59:24.733Z        info    VictoriaMetrics/app/vlstorage/main.go:62        opening storage at -storageDataPath=/vlogs                                                                 
victorialogs  | 2024-06-28T10:59:24.738Z        info    VictoriaMetrics/lib/memory/memory.go:42 limiting caches to 2386516377 bytes, leaving 1591010919 bytes to the OS according to -memory.allowedPercent=60     
victorialogs  | 2024-06-28T10:59:24.786Z        info    VictoriaMetrics/app/vlstorage/main.go:68        successfully opened storage in 0.053 seconds; smallParts: 17; bigParts: 0; smallPartBlocks: 52; bigPartBlocks: 0; smallPartRows: 263; bigPartRows: 0; smallPartSize: 22030 bytes; bigPartSize: 0 bytes                                                                                                                        
victorialogs  | 2024-06-28T10:59:24.787Z        info    VictoriaMetrics/app/victoria-logs/main.go:50    started VictoriaLogs in 0.054 seconds; see https://docs.victoriametrics.com/victorialogs/                  
victorialogs  | 2024-06-28T10:59:24.787Z        info    VictoriaMetrics/lib/httpserver/httpserver.go:119        starting server at http://127.0.0.1:9428/                                                          
victorialogs  | 2024-06-28T10:59:24.787Z        info    VictoriaMetrics/lib/httpserver/httpserver.go:120        pprof handlers are exposed at http://127.0.0.1:9428/debug/pprof/                                   
                                                                                                                                                                                                                   
                                                                                                                                                                                                                   
                                                                                                                                                                                                                   
                                                                                                                                                                                                                   
victorialogs  | panic: runtime error: index out of range [0] with length 0                                                                                                                                         
victorialogs  |                                                                                                                                                                                                    
victorialogs  | goroutine 5 [running]:                                                                                                                                                                             
victorialogs  | github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog.(*syslogLineReader).nextLine(0x400015ed50)                                                                                          
victorialogs  |         github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog/syslog.go:424 +0x884                                                                                                        
victorialogs  | github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog.processUncompressedStream({0x7f40e12818?, 0x4000092280?}, {0xb281a0, 0x40000922b0})                                                 
victorialogs  |         github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog/syslog.go:360 +0xbc                                                                                                         
victorialogs  | github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog.processStreamInternal({0x7f40e12818?, 0x4000092280?}, {0x0, 0x0}, {0xb281a0?, 0x40000922b0?})                                       
victorialogs  |         github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog/syslog.go:337 +0x1e4                                                                                                        
victorialogs  | github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog.processStream({0x7f40e12818, 0x4000092280}, {0x0, 0x0}, 0x40000fc080)                                                               
victorialogs  |         github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog/syslog.go:312 +0xc0                                                                                                         
victorialogs  | github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog.serveUDP.func1()                                                                                                                    
victorialogs  |         github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog/syslog.go:251 +0x4a0                                                                                                        
victorialogs  | created by github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog.serveUDP in goroutine 3                                                                                                  
victorialogs  |         github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog/syslog.go:224 +0x90 
```

### To Reproduce

- Run victorialogs through docker compose
```yaml
services:
    victorialogs:
      container_name: victorialogs
      image: docker.io/victoriametrics/victoria-logs:latest
      command:
        - "--loggerLevel=INFO"
        - "--storageDataPath=/vlogs"
        - "--httpListenAddr=:9428"
        - "-syslog.timezone=Europe/Paris"
        - "-syslog.listenAddr.udp=:514"
      volumes:
        - vldata:/vlogs
      ports:
        - "9428:9428"
        - "514:514/udp"
      networks:
        - lan_logs 
networks:
  lan_logs:
volumes:
  vldata: {}
```
- Configure Wifi controller basically:
![Selection_001](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/12083248/60be6c3d-ec79-4ef7-8542-521d4eae35ac)


### Version


victorialogs  | victoria-logs-20240627-130708-tags-v0.24.0-victorialogs-0-g7c8c040502


### Logs

```bash
victorialogs  | panic: runtime error: index out of range [0] with length 0                                                                                                                                         
victorialogs  |                                                                                                                                                                                                    
victorialogs  | goroutine 5 [running]:                                                                                                                                                                             
victorialogs  | github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog.(*syslogLineReader).nextLine(0x400015ed50)                                                                                          
victorialogs  |         github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog/syslog.go:424 +0x884                                                                                                        
victorialogs  | github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog.processUncompressedStream({0x7f40e12818?, 0x4000092280?}, {0xb281a0, 0x40000922b0})                                                 
victorialogs  |         github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog/syslog.go:360 +0xbc                                                                                                         
victorialogs  | github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog.processStreamInternal({0x7f40e12818?, 0x4000092280?}, {0x0, 0x0}, {0xb281a0?, 0x40000922b0?})                                       
victorialogs  |         github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog/syslog.go:337 +0x1e4                                                                                                        
victorialogs  | github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog.processStream({0x7f40e12818, 0x4000092280}, {0x0, 0x0}, 0x40000fc080)                                                               
victorialogs  |         github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog/syslog.go:312 +0xc0                                                                                                         
victorialogs  | github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog.serveUDP.func1()                                                                                                                    
victorialogs  |         github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog/syslog.go:251 +0x4a0                                                                                                        
victorialogs  | created by github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog.serveUDP in goroutine 3                                                                                                  
victorialogs  |         github.com/VictoriaMetrics/VictoriaMetrics/app/vlinsert/syslog/syslog.go:224 +0x90 
```

### Screenshots

_No response_

### Used command-line flags

_No response_

### Additional information

This is the logs (received in a standard rsyslog) should be received by victorialogs when I click on apply in my Wifi controller:
```bash
2024-06-28T13:20:56.499321+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.146: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpListener::handleAccept(ConnectionPtr, ConstErrorCode&)>
2024-06-28T13:20:56.499321+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.146: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::fillAddress()>
2024-06-28T13:20:56.499576+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.146: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::fillAddress()<
2024-06-28T13:20:56.499576+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.146: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void JsonRPCServer::insert(ConnectionPtr)>
2024-06-28T13:20:56.499576+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.146: %CHAGENT-6-CHAGENT_INFO: logger_services.c:3517 [8169][Info]JsonRPCServer add conn from 127.0.0.1:43882
2024-06-28T13:20:56.499576+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.146: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void JsonRPCServer::insert(ConnectionPtr)<
2024-06-28T13:20:56.499576+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.146: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::start()>
2024-06-28T13:20:56.499576+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.146: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]void HttpConnection::startTransfer()>
2024-06-28T13:20:56.500277+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.146: %CHAGENT-6-CHAGENT_INFO: logger_services.c:3517  event: connection up  type: TCP remote ip: 127.0.0.1 remote port: 43882 local ip: 127.0.0.1 local port: 9998 
2024-06-28T13:20:56.500277+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.146: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]void HttpConnection::startTransfer()>
2024-06-28T13:20:56.500277+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.146: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::start()<
2024-06-28T13:20:56.500277+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.146: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpListener::handleAccept(ConnectionPtr, ConstErrorCode&) self:  local IP: 127.0.0.1 local Port: 9998
2024-06-28T13:20:56.500277+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.147: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpListener::handleAccept(ConnectionPtr, ConstErrorCode&) connection:  status: 5 remote IP: 127.0.0.1 remote Port 43882 local IP 127.0.0.1 local Port 9998
2024-06-28T13:20:56.500277+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.147: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]void HttpListener::startAccept()>
2024-06-28T13:20:56.500277+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.147: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]HttpConnection::HttpConnection(Logger&, IoServicePool&, ConnectionControllerPtr, ConnectionUserPtr, TokenizerPtr, std::__cxx11::string)>
2024-06-28T13:20:56.500277+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.148: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]HttpConnection::HttpConnection(Logger&, IoServicePool&, ConnectionControllerPtr, ConnectionUserPtr, TokenizerPtr, std::__cxx11::string)<
2024-06-28T13:20:56.500277+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.148: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]void HttpListener::startAccept()<
2024-06-28T13:20:56.500277+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.148: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpListener::handleAccept(ConnectionPtr, ConstErrorCode&)<
2024-06-28T13:20:56.500277+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.148: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::handleRead(ConstErrorCode&, std::size_t)>
2024-06-28T13:20:56.500277+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.148: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::handleRead(ConstErrorCode&, std::size_t) self:  status: 5 remote IP: 127.0.0.1 remote Port 43882 local IP 127.0.0.1 local Port 9998 read bytes: 214 error: system:0
2024-06-28T13:20:56.500277+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.148: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::handleRead(ConstErrorCode&, std::size_t)length 77data {"jsonrpc":"2.0","method":"log-level.set","params":{"log-level":7},"id":"70"}
2024-06-28T13:20:56.500897+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.148: %CHAGENT-6-CHAGENT_INFO: logger_services.c:3517 [8169][Info]virtual jsonrpc::json jsonrpc::LogLevelSet::operator()(jsonrpc::json) input: {"log-level":7}
2024-06-28T13:20:56.500897+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.148: %CHAGENT-6-CHAGENT_INFO: logger_services.c:3517 agent log level is set to 7
2024-06-28T13:20:56.500897+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.148: %CHAGENT-6-CHAGENT_INFO: logger_services.c:3517 [8169][Info]void JsonRPCServer::sendResp(std::__cxx11::string, ConnectionPtr) send message: {"id":"70","jsonrpc":"2.0","result":{}}
2024-06-28T13:20:56.500897+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.148: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::sendMessage(RawMessagePtr)>
2024-06-28T13:20:56.500897+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.148: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::sendMessage(RawMessagePtr) connection:  status: 5 remote IP: 127.0.0.1 remote Port 43882 local IP 127.0.0.1 local Port 9998
2024-06-28T13:20:56.500897+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.149: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::sendMessage(RawMessagePtr) message: std::ostream& operator<<(std::ostream&, const RawMessage&) length: 39 data: {"id":"70","jsonrpc":"2.0","result":{}}#012
2024-06-28T13:20:56.500897+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.149: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::sendMessage(RawMessagePtr)<
2024-06-28T13:20:56.500897+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.149: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::stop()>
2024-06-28T13:20:56.500897+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.149: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::stop()self:  status: 5 remote IP: 127.0.0.1 remote Port 43882 local IP 127.0.0.1 local Port 9998
2024-06-28T13:20:56.501420+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.149: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8168]virtual void HttpConnection::handleWrite(ConstErrorCode&, std::size_t)>
2024-06-28T13:20:56.501420+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.149: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::stop()<
2024-06-28T13:20:56.501420+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.149: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::handleRead(ConstErrorCode&, std::size_t)>
2024-06-28T13:20:56.501420+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.150: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8168]virtual void HttpConnection::handleWrite(ConstErrorCode&, std::size_t)self:  status: 7 remote IP: 127.0.0.1 remote Port 43882 local IP 127.0.0.1 local Port 9998 write bytes: 135 error: system:0
2024-06-28T13:20:56.501420+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:56.150: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8168]virtual void HttpConnection::handleWrite(ConstErrorCode&, std::size_t)0xb5008de8 send Q is empty 
2024-06-28T13:20:57.555772+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.151: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::handleCloseTimeout(ConstErrorCode&)>
2024-06-28T13:20:57.555772+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.151: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::handleCloseTimeout(ConstErrorCode&)<
2024-06-28T13:20:57.556168+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.152: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::handleRead(ConstErrorCode&, std::size_t)>
2024-06-28T13:20:57.556168+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.152: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::handleRead(ConstErrorCode&, std::size_t) self:  status: 7 remote IP: 127.0.0.1 remote Port 43882 local IP 127.0.0.1 local Port 9998 read bytes: 0 error: system:125
2024-06-28T13:20:57.556168+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.152: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void HttpConnection::handleRead(ConstErrorCode&, std::size_t) operation_aborted. current status: 7
2024-06-28T13:20:57.556168+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.152: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]void HttpConnection::handleError(ConstErrorCode&)>
2024-06-28T13:20:57.556168+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.152: %CHAGENT-6-CHAGENT_INFO: logger_services.c:3517 [8169][Info]jsonrpc server connection ec: asio.misc:2
2024-06-28T13:20:57.556168+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.152: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void JsonRPCServer::erase(ConnectionPtr)>
2024-06-28T13:20:57.556168+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.152: %CHAGENT-6-CHAGENT_INFO: logger_services.c:3517 [8169][Info]JsonRPCServer del conn from 127.0.0.1:43882
2024-06-28T13:20:57.556168+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.152: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual void JsonRPCServer::erase(ConnectionPtr)<
2024-06-28T13:20:57.556168+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.152: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]void HttpConnection::handleError(ConstErrorCode&)<
2024-06-28T13:20:57.556168+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.153: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual HttpConnection::~HttpConnection()>
2024-06-28T13:20:57.556168+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.153: %CHAGENT-6-CHAGENT_INFO: logger_services.c:3517  event: connection down  type: TCP remote ip: 127.0.0.1 remote port: 43882 local ip: 127.0.0.1 local port: 9998 
2024-06-28T13:20:57.556168+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.153: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]virtual HttpConnection::~HttpConnection()<
2024-06-28T13:20:57.556168+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.153: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]size_t IoServicePool::gClear() 1 garbage Connection cleared
2024-06-28T13:20:57.556168+02:00 192.168.2.148 ap-garage: *ProcessLoggingTask: Jun 28 13:20:57.153: %CHAGENT-7-CHAGENT_DEBUG: logger_services.c:3517 [8169]void IoServicePool::run()will reset
```

**Comment by f41gh7** [comment]
Thanks for reporting

**Comment by valyala** [comment]
The issue should be fixed in the commit d7185f1b7704d29d66d4ffad5ea7c50dfa106b27 . This commit will be included in the next release.

**Comment by valyala** [comment]
The bugfix for this issue has been included in [VictoriaLogs v0.25.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v0.25.0-victorialogs). Closing the issue as fixed.

**Comment by gesnaud** [comment]
Hi @valyala !

Nice job! Everything is working like a charm!

Good job 👍

Thanks!
